### PR TITLE
my work is done and ready to be reviewed

### PR DESCRIPTION
**Pull request by chidinmaamara**
@promise-J, my code is ready to be reviewed

### CODE REVIEW

File: html/about.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (promise-J) [inline html/about.html]: @chidinmaamara , your title in about.html page is not good. please give it a more descriptive name

File: html/index.html [hunk 2]
```diff
@@ -6,9 +6,10 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="../css/global.css" />
     <link rel="stylesheet" href="../css/style.css" />
-    <title>Document</title>
+    <title>homepage</title>
   </head>
   <body>
     <h1>Welcome To LM Techub!</h1>
+    <p>This is a silly line</p>
```
Comment (promise-J) [inline html/index.html:13]: I dont get what you are doing. can you more seriouser

**Review by promise-J — CHANGES_REQUESTED** [review]
You have to have a sense of imagination to know what you should do to the lint

File: html/index.html (hunk 2 above)
Comment (chidinmaamara) [reply html/index.html:13]: what do you mean by seriouser?

**Review by promise-J — APPROVED** [review]
Congratulations!!!! You are doing well.

**Pull request merged**